.meteor/packages

```diff
@@ -24,3 +24,4 @@ aldeed:simple-schema
 aldeed:collection2
 aldeed:autoform
 twbs:bootstrap
+lukemadera:autoform-googleplace
```

Comment:
Go ahead and commit your changes to the packages and versions files after removing the `lukemadera:autoform-googleplace` package.